### PR TITLE
Change route_gateway and route_cidr to computed

### PR DESCRIPTION
**Pull request by futuretea**
Change route_gateway and route_cidr to computed to ignore route_gateway and route_cidr changes
**Related issues**
https://github.com/harvester/harvester/issues/2261

### CODE REVIEW

File: internal/provider/network/schema_network.go [hunk 1]
```diff
@@ -39,12 +39,14 @@ func Schema() map[string]*schema.Schema {
 		constants.FieldNetworkRouteCIDR: {
 			Type:          schema.TypeString,
 			Optional:      true,
+			Computed:      true,
```
Comment (guangbochen) [inline internal/provider/network/schema_network.go:42]: IIRC, both CIDR and gateway can be configured by the users, so computed may not be a valid solution.
https://www.terraform.io/plugin/sdkv2/schemas/schema-behaviors#computed

Comment (futuretea) [reply internal/provider/network/schema_network.go:42]: Users can set value to a Computed field

Comment (guangbochen) [reply internal/provider/network/schema_network.go:42]: Does it mean the `computed` definition can help to merge the output after `tf apply` and it is also configurable if needed?

Comment (futuretea) [reply internal/provider/network/schema_network.go:42]: Yes

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**